etc/docker/elyra/Dockerfile

```diff
@@ -32,7 +32,8 @@ RUN chmod ugo+x /usr/local/bin/start-elyra.sh && \
 USER $NB_USER
 
 RUN python -m pip install --upgrade pip && \
-    python -m pip install setuptools pandas --ignore-installed --upgrade && \
+    python -m pip install --ignore-installed --upgrade setuptools && \
+    python -m pip install --upgrade pandas numpy papermill nbclient jupyter-client notebook  && \
```

Comment:
Why do we have `pandas` and `numpy` here?

Reply:
pandas was there, numpy usually needs to be sync-ed up with pandas